crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -296,34 +244,16 @@ impl RefreshTask {
             ),
         );
 
-        let ctx = SessionContext::new();
+        let record_batch_stream = Box::pin(observed_record_batch_stream);
+        let sink_lock = self.sink.read().await;
+        let sink = &*sink_lock;
```

Comment:
Instead of inserting directly into the acceleration TableProvider, we now have a new `AccelerationSink` concept that can handle multiplexing the insertion stream across multiple tables